app/helpers/validation_request_helper.rb

```diff
@@ -58,6 +58,14 @@ def cancel_request_url(planning_application, validation_request)
          validation_request)
   end
 
+  def edit_validation_request_url(validation_request)
+    link_to "Edit",
+            send("edit_planning_application_#{request_type(validation_request)}_path", validation_request.planning_application,
+                 validation_request)
+  rescue NoMethodError
+    nil
+  end
+
```

Comment:
note: this is so document replacements just have an empty string under the column instead of "Edit": 
![image](https://user-images.githubusercontent.com/9452321/147766412-7511904b-53c9-45c1-aa05-a878e436d2e9.png)
had tried leaving the rescue empty but rubocop complained so we're returning nil instead.